### DISCUSSION TITLE
How can I put a gap on a graph?

### DISCUSSION THREAD

**Discussion by yxsjune** [leonhma/obsidian-functionplot#18] [Q&A]
I can't find a way to add gaps on the graph.

**Comment by leonhma** [comment]
What do you mean by 'gap'? 

Edit: obsidian-functionplot is strictly for plotting functions. If you are looking to represent data measurements, take a look at [obsidian-charts](https://charts.phibr0.de/).

**Reply by yxsjune** [reply]
What I meant by gap is [this](https://www.google.com/search?q=gap+discontinuity&tbm=isch&hl=en-us&client=safari&prmd=isnv&sa=X&ved=2ahUKEwi2-vGOqK_5AhUJ8ZQKHdY-B3oQrNwCKAB6BQgBENAB&biw=414&bih=715#imgrc=DUkdRghYvran4M). Sorry for unclear description.
One more question: are there any ways to mark a specific point on the coordinate plane?

**Reply by leonhma** [reply]
So you mean something like 1/x? Because it draws this just fine.

The annotations feature exists in the library I am using to draw the plots. I can probably even add this feature today and release a new version.

**Reply by yxsjune** [reply]
No, I meant the circle used to describe that the function is discontinuous. I will be so grateful if you add annotations features. I’m using your plugin for studying math and it’s been really helpful.

**Reply by leonhma** [reply]
I am also using the plugin for taking notes in school, but it's 10th grade, so I gotta look into discontinuities a bit more.

By annotations do you mean marking a dot and having a label for it, or do you mean drawing a line (`lim`, etc.) with a label?

**Reply by yxsjune** [reply]
I got it. Discontinuity is a concept used in calculus. How about piecewise functions? Is plotting a piecewise function available in this plugin?

I meant the former one.

**Reply by leonhma** [reply]
No there currently is no functionality like that. It basically just does what it says in the readme. I'll look into it and possibilities for integrating that into the syntax.

I don't think the library I am using does annotated points so I'll have to submit a pull request there too.

I'll open some issues to organize progress and move the conversation not there

**Reply by leonhma** [reply]
"...move the conversation **to** there" of course.

**Reply by yxsjune** [reply]
Thanks for your effort. Keep up with it!
